MVP Module/Source/Feed Presentation/FeedPresenter.swift

```diff
@@ -32,6 +39,7 @@ final class FeedPresenter {
 	}
 
 	func didFinishLoadingFeed(with error: Error) {
+		feedView.display(FeedErrorViewModel(errorMessage: Localized.Feed.loadError))
```

Comment:
Same as my previous comment:
```suggestion
		errorView.display(FeedErrorViewModel(errorMessage: Localized.Feed.loadError))
```

Reply:
done